kahuna/public/js/services/api/leases.js

```diff
@@ -0,0 +1,70 @@
+import angular from 'angular';
+import Rx from 'rx';
+
+import './media-api';
+import {service} from '../../edits/service';
+
+var leaseService = angular.module('kahuna.services.lease', [
+  service.name
+  ]);
+
+leaseService.factory('leaseService', [
+  '$rootScope',
+  '$q',
+  'imageAccessor',
+  'mediaApi',
+  'editsService',
+  function ($rootScope, $q, imageAccessor, mediaApi, editsService) {
+    var leasesRoot;
```

Comment:
yea not the prettiest... took the pattern from here: https://github.com/guardian/grid/blob/master/kahuna/public/js/services/api/media-cropper.js

open to improvements though